lib/shared/models/selection-state.js

```diff
@@ -21,49 +21,49 @@ const store = require('../store')
 const availableDrives = require('./available-drives')
 
 /**
- * @summary Set a drive
+ * @summary Select a drive by its device path
  * @function
  * @public
  *
- * @param {String} drive - drive device
+ * @param {String} driveDevice - drive device
  *
  * @example
- * selectionState.setDrive('/dev/disk2');
+ * selectionState.selectDrive('/dev/disk2');
  */
-exports.setDrive = (drive) => {
+exports.selectDrive = (driveDevice) => {
   store.dispatch({
     type: store.Actions.SELECT_DRIVE,
-    data: drive
+    data: driveDevice
   })
 }
 
 /**
- * @summary Toggle set drive
+ * @summary Toggle drive selection
  * @function
  * @public
  *
- * @param {String} drive - drive device
+ * @param {String} driveDevice - drive device
  *
  * @example
- * selectionState.toggleSetDrive('/dev/disk2');
+ * selectionState.toggleDrive('/dev/disk2');
  */
-exports.toggleSetDrive = (drive) => {
-  if (exports.isCurrentDrive(drive)) {
-    exports.removeDrive()
+exports.toggleDrive = (driveDevice) => {
+  if (exports.isCurrentDrive(driveDevice)) {
+    exports.unselectDrive()
```

Comment:
Can we use "deselect" here ("unselect" isn't to be found in dictionaries)?

Reply:
Done